python/doc/theory/meta_modeling/chaos_basis.rst

```diff
@@ -3,132 +3,138 @@
 Polynomial chaos basis
 ----------------------
 
-| The current section is focused on a specific kind of functional chaos
-  representation that has been implemented, namely
-  *polynomial chaos expansions*.
+The current section is focused on a specific kind of functional chaos
+representation that has been implemented, namely
+*polynomial chaos expansions*.
 
-| **Mathematical framework**
+Mathematical framework
+~~~~~~~~~~~~~~~~~~~~~~
 
-| Throughout this section, the model response is assumed to be a
-  *scalar* random variable :math:`Y = h(\underline{X})`. However, the
-  following derivations hold in case of a vector-valued response.
-| Let us suppose that:
+Throughout this section, the model response is assumed to be a
+*scalar* random variable :math:`Y = h(\underline{X})`. However, the
```

Comment:
\underline should be replaced with \vect here and elsewhere

Reply:
That is true: fixed at da97ad38057f199069c2d3d02accf57cb2542d30.